test_requests_kerberos.py

```diff
@@ -211,24 +211,6 @@ def test_authenticate_server(self):
             self.assertTrue(result)
             clientStep_complete.assert_called_with("CTX", "servertoken")
 
-    def test_handle_other(self):
```

Comment:
Why does this need to be deleted? It seems handle_other was renamed, why not  rename things here appropriately?